<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-L9I71BEA" name="com.lima.test.AdjunoLIMADocumentReceivedTest" tests="21" failures="3" timestamp="2 Feb 2018 12:31:25 GMT" time="304.602" errors="2">
  <testcase name="test_16_2_SetFieldMultipleValue" time="43.456" classname="com.lima.test.AdjunoLIMADocumentReceivedTest"/>
  <testcase name="clickHyperlink" time="20.464" classname="com.lima.test.AdjunoLIMADocumentReceivedTest">
    <error type="org.openqa.selenium.NoSuchElementException" message="Unable to locate element: {&quot;method&quot;:&quot;xpath&quot;,&quot;selector&quot;:&quot;.//*[@id=&#039;ext-gen185&#039;]/div[1]/table/tbody/tr/td[4]/div/a&quot;}
Command duration or timeout: 10.24 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;2.53.1&#039;, revision: &#039;a36b8b1&#039;, time: &#039;2016-06-30 17:32:46&#039;
System info: host: &#039;LAPTOP-L9I71BEA&#039;, ip: &#039;192.168.1.16&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_121&#039;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: f6c23f48-4848-433a-bcfc-e039c76ef6da
*** Element info: {Using=xpath, value=.//*[@id=&#039;ext-gen185&#039;]/div[1]/table/tbody/tr/td[4]/div/a}">
      <![CDATA[org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":".//*[@id='ext-gen185']/div[1]/table/tbody/tr/td[4]/div/a"}
Command duration or timeout: 10.24 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:32:46'
System info: host: 'LAPTOP-L9I71BEA', ip: '192.168.1.16', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_121'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: f6c23f48-4848-433a-bcfc-e039c76ef6da
*** Element info: {Using=xpath, value=.//*[@id='ext-gen185']/div[1]/table/tbody/tr/td[4]/div/a}
	at sun.reflect.GeneratedConstructorAccessor23.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:363)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:500)
	at org.openqa.selenium.By$ByXPath.findElement(By.java:361)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:355)
	at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
	at com.sun.proxy.$Proxy9.click(Unknown Source)
	at com.lima.test.AdjunoLIMADocumentReceivedTest.clickHyperlink(AdjunoLIMADocumentReceivedTest.java:606)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
Caused by: org.openqa.selenium.NoSuchElementException: Unable to locate element: {"method":"xpath","selector":".//*[@id='ext-gen185']/div[1]/table/tbody/tr/td[4]/div/a"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:32:46'
System info: host: 'LAPTOP-L9I71BEA', ip: '192.168.1.16', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_121'
Driver info: driver.version: unknown
	at <anonymous class>.FirefoxDriver.prototype.findElementInternal_(file:///C:/Users/Anand/AppData/Local/Temp/anonymous2592470802840224872webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:10770)
	at <anonymous class>.fxdriver.Timer.prototype.setTimeout/<.notify(file:///C:/Users/Anand/AppData/Local/Temp/anonymous2592470802840224872webdriver-profile/extensions/fxdriver@googlecode.com/components/driver-component.js:625)
]]>
    </error>
  </testcase> <!-- clickHyperlink -->
  <testcase name="checkforValidationBox" time="0.094" classname="com.lima.test.AdjunoLIMADocumentReceivedTest"/>
  <testcase name="checkforchevrons" time="0.875" classname="com.lima.test.AdjunoLIMADocumentReceivedTest"/>
  <testcase name="test_19d1_CompleteChev" time="31.031" classname="com.lima.test.AdjunoLIMADocumentReceivedTest"/>
  <testcase name="verifyDocumentReceivedFields" time="1.437" classname="com.lima.test.AdjunoLIMADocumentReceivedTest"/>
  <testcase name="test_17d7_UpdateTrackAWB" time="19.188" classname="com.lima.test.AdjunoLIMADocumentReceivedTest"/>
  <testcase name="verifyFileNameAndSize" time="10.236" classname="com.lima.test.AdjunoLIMADocumentReceivedTest"/>
  <testcase name="Test_close" time="25.498" classname="com.lima.test.AdjunoLIMADocumentReceivedTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	close is not available expected [true] but found [false],
	*****_13_5_close is not clicked in pop up***** expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	close is not available expected [true] but found [false],
	*****_13_5_close is not clicked in pop up***** expected [true] but found [false]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at com.lima.test.AdjunoLIMADocumentReceivedTest.Test_close(AdjunoLIMADocumentReceivedTest.java:721)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </failure>
  </testcase> <!-- Test_close -->
  <testcase name="test_16d1_AccessDocumentReceived" time="29.180" classname="com.lima.test.AdjunoLIMADocumentReceivedTest"/>
  <testcase name="attachmentDocStatus" time="10.205" classname="com.lima.test.AdjunoLIMADocumentReceivedTest"/>
  <testcase name="clickDeleteDocumentBtn" time="7.907" classname="com.lima.test.AdjunoLIMADocumentReceivedTest"/>
  <testcase name="checkuploadfile" time="13.752" classname="com.lima.test.AdjunoLIMADocumentReceivedTest">
    <error type="org.openqa.selenium.ElementNotVisibleException" message="Element is not currently visible and so may not be interacted with
Command duration or timeout: 10.22 seconds
Build info: version: &#039;2.53.1&#039;, revision: &#039;a36b8b1&#039;, time: &#039;2016-06-30 17:32:46&#039;
System info: host: &#039;LAPTOP-L9I71BEA&#039;, ip: &#039;192.168.1.16&#039;, os.name: &#039;Windows 10&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;10.0&#039;, java.version: &#039;1.8.0_121&#039;
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: f6c23f48-4848-433a-bcfc-e039c76ef6da">
      <![CDATA[org.openqa.selenium.ElementNotVisibleException: Element is not currently visible and so may not be interacted with
Command duration or timeout: 10.22 seconds
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:32:46'
System info: host: 'LAPTOP-L9I71BEA', ip: '192.168.1.16', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_121'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=47.0.1, platform=WINDOWS, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: f6c23f48-4848-433a-bcfc-e039c76ef6da
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:327)
	at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:85)
	at sun.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
	at com.sun.proxy.$Proxy9.click(Unknown Source)
	at com.lima.pagefactory.AdjunoLIMADocumentReceivedPOM.clickButtonUsingWebElement(AdjunoLIMADocumentReceivedPOM.java:966)
	at com.lima.test.AdjunoLIMADocumentReceivedTest.checkuploadfile(AdjunoLIMADocumentReceivedTest.java:639)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
Caused by: org.openqa.selenium.ElementNotVisibleException: Element is not currently visible and so may not be interacted with
Build info: version: '2.53.1', revision: 'a36b8b1', time: '2016-06-30 17:32:46'
System info: host: 'LAPTOP-L9I71BEA', ip: '192.168.1.16', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_121'
Driver info: driver.version: unknown
	at <anonymous class>.fxdriver.preconditions.visible(file:///C:/Users/Anand/AppData/Local/Temp/anonymous2592470802840224872webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:10092)
	at <anonymous class>.DelayedCommand.prototype.checkPreconditions_(file:///C:/Users/Anand/AppData/Local/Temp/anonymous2592470802840224872webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12644)
	at <anonymous class>.DelayedCommand.prototype.executeInternal_/h(file:///C:/Users/Anand/AppData/Local/Temp/anonymous2592470802840224872webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12661)
	at <anonymous class>.fxdriver.Timer.prototype.setTimeout/<.notify(file:///C:/Users/Anand/AppData/Local/Temp/anonymous2592470802840224872webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:625)
]]>
    </error>
  </testcase> <!-- checkuploadfile -->
  <testcase name="uploadmultiplefile" time="3.203" classname="com.lima.test.AdjunoLIMADocumentReceivedTest"/>
  <testcase name="test_17d5_UpdateChevValidation" time="32.995" classname="com.lima.test.AdjunoLIMADocumentReceivedTest"/>
  <testcase name="test_16d1_SetFeildDocumentReceivedfield" time="25.901" classname="com.lima.test.AdjunoLIMADocumentReceivedTest"/>
  <testcase name="checkforValidationMsg" time="0.235" classname="com.lima.test.AdjunoLIMADocumentReceivedTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	error msg is not Visible expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	error msg is not Visible expected [true] but found [false]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at com.lima.test.AdjunoLIMADocumentReceivedTest.checkforValidationMsg(AdjunoLIMADocumentReceivedTest.java:594)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </failure>
  </testcase> <!-- checkforValidationMsg -->
  <testcase name="clickAddButton" time="2.938" classname="com.lima.test.AdjunoLIMADocumentReceivedTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Clear All Button is not Present expected [true] but found [false],
	Remove Button is not Present expected [true] but found [false],
	Upload Button is not Present expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	Clear All Button is not Present expected [true] but found [false],
	Remove Button is not Present expected [true] but found [false],
	Upload Button is not Present expected [true] but found [false]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at com.lima.test.AdjunoLIMADocumentReceivedTest.clickAddButton(AdjunoLIMADocumentReceivedTest.java:630)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </failure>
  </testcase> <!-- clickAddButton -->
  <testcase name="clickadddocumentbtn" time="1.734" classname="com.lima.test.AdjunoLIMADocumentReceivedTest"/>
  <testcase name="ClearDocumentReceivedFields" time="20.788" classname="com.lima.test.AdjunoLIMADocumentReceivedTest"/>
  <testcase name="selectCheckBoxData" time="3.485" classname="com.lima.test.AdjunoLIMADocumentReceivedTest"/>
</testsuite> <!-- com.lima.test.AdjunoLIMADocumentReceivedTest -->
